examples/getting_started.ipynb

```diff
@@ -137,7 +185,7 @@
    "id": "1e8a772d",
    "metadata": {},
    "source": [
-    "Then, we can simply print the MetaFrame to display it in an easy-to-read format:"
+    "We can use the built-in Python `print` function to display the (statistical metadata contained in the) MetaFrame in an easy-to-read format:"
```

Comment:
Normally I would agree, but due to this being a tutorial I like the extra information.

Reply:
What about then just removing the parentheses?

Reply:
Or maybe reformulate it a bit to make it more descriptive? I think the reason I fell over it is also because I don't know if "statistical metadata" will really ring a bell with the audience anyway. Well, it can always be improved at a later date I suppose!